templates/single.twig

```diff
@@ -1,16 +1,16 @@
 {% extends 'base.twig' %}
 
 {% block content %}
-  <!-- Provided by templates/single.twig -->
-  {% embed '@layouts/content/content.twig' with {
-    'has_constrain': true
-  } %}
-    {% block content %}
-      {% if function( 'get_post_format' ) %}
-        {% include 'post-formats/content-' ~ function( 'get_post_format' ) ~ '.twig' %}
-      {% else %}
-        {% include 'post-formats/content.twig' %}
-      {% endif %}
-    {% endblock %}
-  {% endembed %}
```

Comment:
Similar to other PR, please set your editor to use 2 spaces instead of 4 spaces or tabs.